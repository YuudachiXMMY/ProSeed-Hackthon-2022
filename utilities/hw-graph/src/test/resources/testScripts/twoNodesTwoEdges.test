# Test to add two nodes with edges between them

# Create a graph
CreateGraph graph2

# Add two nodes to graph
AddNode graph2 node1
AddNode graph2 node2

# Add two edges to graph
AddEdge graph2 node1 node2 edge1
AddEdge graph2 node2 node1 edge2

# Print all nodes in graph
ListNodes graph2
ListChildren graph2 node1
ListChildren graph2 node2