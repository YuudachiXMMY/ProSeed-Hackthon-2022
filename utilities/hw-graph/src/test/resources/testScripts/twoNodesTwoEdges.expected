# Test to add two nodes with edges between them

# Create a graph
created graph graph2

# Add two nodes to graph
added node node1 to graph2
added node node2 to graph2

# Add two edges to graph
added edge edge1 from node1 to node2 in graph2
added edge edge2 from node2 to node1 in graph2

# Print all nodes in graph
graph2 contains: node1 node2
the children of node1 in graph2 are: node2(edge1)
the children of node2 in graph2 are: node1(edge2)